scripts/deploy_on_kind.sh

```diff
@@ -33,6 +36,13 @@ else
 fi
 
 kubectl apply -k manifests/kustomize/overlays/db
-kubectl set image -n kubeflow deployment/model-registry-deployment rest-container="$IMG"
+kubectl patch deployment -n "$MR_NAMESPACE" model-registry-deployment \
+--patch '{"spec": {"template": {"spec": {"containers": [{"name": "rest-container", "image": "'$IMG'", "imagePullPolicy": "IfNotPresent"}]}}}}'
+
+kubectl delete pod -n "$MR_NAMESPACE" --selector='component=model-registry-server'
+
+repeat_cmd_until "kubectl get pod -n "$MR_NAMESPACE" --selector='component=model-registry-server' \
+-o jsonpath=\"{.items[*].spec.containers[?(@.name=='rest-container')].image}\"" "= $IMG" 300 "kubectl describe pod -n $MR_NAMESPACE --selector='component=model-registry-server'"
+
 kubectl wait --for=condition=available -n "$MR_NAMESPACE" deployment/model-registry-db --timeout=5m
```

Comment:
I think this line (waiting for DB) could go before the `repeat_cmd_until ...` imho (we know MR won't be available at least until mysql is available)